### PR TITLE
Migrations: 1

### PR DESCRIPTION
**Pull request by akshay1y2**

### CODE REVIEW

**Comment by akshay1y2** [comment]
https://docs.google.com/document/d/17GD0udHhXzTmRNOZZy_nsysIBnafIomr_xi4OkL-jYQ/view#

File: db/migrate/20200311065850_add_details_to_products.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class AddDetailsToProducts < ActiveRecord::Migration[6.0]
+  def change
+    add_column :products, :enabled, :boolean, default: false
```
Comment (siddharth28) [inline db/migrate/20200311065850_add_details_to_products.rb]: @akshay1y2 why are we providing a default value here?

File: db/migrate/20200311080401_change_enabled_to_boolean.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class ChangeEnabledToBoolean < ActiveRecord::Migration[6.0]
+  def change
```
Comment (siddharth28) [inline db/migrate/20200311080401_change_enabled_to_boolean.rb]: @akshay1y2 is this required?

File: public/javascripts/translations.js [hunk 3]
```diff
@@ -1,3 +1,3 @@
...
..."","precision":3,"significant":true,"strip_insignificant_zeros":true},"storage_units":{"format":"%n %u","units":{"byte":{"one":"Byte","other":"Bytes"},"eb":"EB","gb":"GB","kb":"KB","mb":"MB","pb":"PB","tb":"TB"}}},"nth":{"ordinalized":{},"ordinals":{}},"percentage":{"format":{"delimiter":"","format":"%n%"}},"precision":{"format":{"delimiter":""}}},"orders":{"form":{"address_html":"Address","check_pay_type":{"account_number":"Account #","routing_number":"Routing #"},"credit_card_pay_type":{"cc_number":"CC #","expiration_date":"Expiry"},"email":"E-mail","name":"Name","pay_prompt_html":"Select a payment method","pay_type":"Pay with","pay_types":{"check":"Check","credit_card":"Credit Card","purchase_order":"Purchase Order"},"purchase_order_pay_type":{"po_number":"PO #"},"submit":"Place Order"},"new":{"legend":"Please Enter Your Details"}},"store":{"index":{"add_html":"Add to Cart","title_html":"The Pragmatic Bookshelf"}},"support":{"array":{"last_word_connector":", and ","two_words_connector":" and ","words_connector":", "}},"thanks":"Thank you for your order","time":{"am":"am","formats":{"default":"%a, %d %b %Y %H:%M:%S %z","long":"%B %d, %Y %H:%M","short":"%d %b %H:%M"},"pm":"pm"}});
```
Comment (siddharth28) [inline public/javascripts/translations.js]: @akshay1y2 don't commit unnecessary code.

**Pull request merged**